scenario/elasticcreateapp.go

```diff
@@ -34,6 +37,24 @@ func (settings ElasticCreateAppSettings) Execute(sessionState *session.State, ac
 		return
 	}
 
+	if !settings.IgnoreEvents {
+		events := sessionState.EventWebsocket()
+		if events == nil {
+			actionState.AddErrors(errors.New("Could not get events websocket"))
+			return
+		}
+
+		ctx, cancel := context.WithCancel(sessionState.BaseContext())
```

Comment:
Should probably add a large default (and definable) timeout so we don't wait in this action forever if event never arrives. (same as should be added to other actions waiting for events). In this case maybe even just using the `timeout` setting from settings (i.e. the other context function from sessionstate) since this event should always be rather quick (other event waiters should use their own settings i.e. reload could take very long). If not received event before timeout -> report error